src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-flirpy.py

```diff
@@ -0,0 +1,19 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
```

Comment:
Ah, missed this one earlier - should be 2021.

Reply:
I'll get that updated too.  Does the Copyright in test_libraries.py also need to be bumped to 2021?

Reply:
> Does the Copyright in test_libraries.py also need to be bumped to 2021?

Actually, it looks like all files in this repository will need a copyright year update - it was done in the main repository, but apparently not here yet...

So for this PR, I'd just fix it in the new hook file, while the rest will be addressed separately.